rtcdelta.py

```diff
@@ -29,22 +31,50 @@ def ensure_path_exists(path):
     if not os.path.exists(path):
         os.mkdir(path)
 
+
+def get_changes(changesets):
+    if not changesets:
+        return
+
+    for changeset in changesets:
```

Comment:
I believe we need to sort the change sets first so we get the changes in chronological order. Otherwise we will fetch the BEFORE and AFTER in an incorrect order, no? (This problem is not exclusive to your PR, so maybe we should address it as a separate issue ticket - I will do that)